include/exadg/structure/time_integration/time_int_gen_alpha.cpp

```diff
@@ -120,13 +120,27 @@ TimeIntGenAlpha<dim, Number>::do_timestep_solve()
   dealii::Timer timer;
   timer.restart();
 
-  // compute const_vector
-  VectorType const_vector, rhs;
+  // add acceleration term remainder to rhs
+  VectorType tmp;
+  tmp.reinit(displacement_n);
+  this->compute_const_vector_acceleration_remainder(tmp,
+                                                    displacement_n,
+                                                    velocity_n,
+                                                    acceleration_n);
+
+  VectorType const_vector;
   const_vector.reinit(displacement_n);
-  rhs.reinit(displacement_n);
-  this->compute_const_vector(rhs, displacement_n, velocity_n, acceleration_n);
-  pde_operator->apply_mass_operator(const_vector, rhs);
+  pde_operator->apply_mass_operator(const_vector, tmp);
 
+  // add weak damping term remainder to rhs
+  if(this->param.weak_damping_active)
+  {
+    this->compute_const_vector_velocity_remainder(tmp, displacement_n, velocity_n, acceleration_n);
```

Comment:
Maybe also here one should change the naming towards `const_vector_acceleration_term` and `const_vector_damping_term`?

Reply:
The current naming is rather general and also explains exactly where it comes from. This was not the case before, but previously this was not an issue, since there was no confusion with other contributions to the mass operator scaling. Now, however, we have multiple contributions and may even have more in the future. I am not sure that there is no other weak form (maybe from viscoelasticity?) that could not introduce the same or very similar term with a different scaling, and the current naming (as "chunky" as it is) in my opinion reflects pretty well where terms and vectors are coming from and what they are. Hence, I would actually stick to the current naming.